model-client/src/commonMain/kotlin/org/modelix/model/oauth/ModelixOAuthClient.kt

```diff
@@ -0,0 +1,67 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.modelix.model.oauth
+
+import io.ktor.client.HttpClientConfig
+import io.ktor.client.plugins.auth.Auth
+import io.ktor.client.plugins.auth.providers.BearerTokens
+import io.ktor.client.plugins.auth.providers.bearer
+
+/**
+ * Functions and states for authenticating to a model server.
+ * Configuration differs for JS and JVM.
+ */
+expect object ModelixOAuthClient {
+    /**
+     * Function to configure the OAuth for an HTTP client.
```

Comment:
the oauth what? Looks like a noun is missing?

Reply:
...

While fixing it, I noticed that only the JVM specific-implemenation uses OAuth.
The common implementation just use generic Bearer authentication 😆 

I updated the comment and names accordingly.